### PR TITLE
added backing canvas for overall tooltip hit rate

### PR DESCRIPTION
**Pull request by ChristianLohmann**
added backing canvas for overall tooltip hit rate

### CODE REVIEW

File: app/src/bars.js [hunk 1]
```diff
@@ -34,35 +39,33 @@ export default class Bars extends PIXI.Graphics {
...
-            for (subKey of Object.keys(bars.resources.pods)) {
-                subText += '\t\t' + subKey + ': ' + bars.resources.pods[subKey] + '\n'
-            }
-            s += subText;
-            bars.tooltip.text.text = s;
-            bars.tooltip.x = bars.toGlobal(new PIXI.Point(0, 0)).x;
-            bars.tooltip.y = bars.toGlobal(new PIXI.Point(0, 0)).y;
+            let s = 'CPU: \n'
+            const {capacity: cpuCap, requested: cpuReq, used: cpuUsed} = bars.resources.cpu
+            s += '\t\t Capactiy  : ' + cpuCap + '\n'
```
Comment (hjacobs) [inline app/src/bars.js]: small typo here

**Comment by hjacobs** [comment]
Can you resolve the conflicts?

**Comment by ChristianLohmann** [comment]
solved conflicts

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**